components/servo/webview_delegate.rs

```diff
@@ -247,6 +247,7 @@ pub trait WebViewDelegate {
 
     /// Open dialog to select bluetooth device.
     /// TODO: This API needs to be reworked to match the new model of how responses are sent.
+    #[cfg(feature = "bluetooth")]
```

Comment:
I think the API bits don't necessarily need to be behind a flag. In general, I think we should avoid changing the API surface depending on build flags as this makes the compilation of embedder applications trickier for embedders. So if it's okay let's expose this and other feature-dependent APIs unconditionally.